spec/controllers/automated_tests_controller_spec.rb

```diff
@@ -378,9 +378,23 @@
                              token_start_date: 1.day.ago,
                              remote_autotest_settings_id: 1)
         end
+        context 'and at least one test-group can be run by students' do
+          let(:assignment) { create :assignment_with_test_groups_student_runnable }
+          # we need to "refresh" the :params variable, which was originally defined at the 'describe' level
+          let(:params) { { course_id: assignment.course.id, assignment_id: assignment.id } }
```

Comment:
This shouldn't be necessary, `params` is lazily-evaluated so should use the definition of `assignment` that's been overridden. (Same with the context block below)